### ISSUE TITLE
[Terminal >Set tab color]: Accessible name is not descriptive for 'Reset' button.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#12044]
### Windows Terminal version

1.12.3472.0

### Windows build number

1.12.3472.0

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Activate 'Actions' tab and then Add a new actions 'Set tab color' with shortcut key 'Ctrl + Shift + c'.
4. Open Narrator using 'Win + Ctrl + Enter' keys.
5. navigate inside Terminal window.
6. Press 'Ctrl + Shift + c' keys, then navigate to 'Reset' control inside Set tab color pop-up using tab key.
7. Observe the issues.

**User Experience:**
Screen Reader users UX will not be good if control name is not descriptive.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[Accessible name is not descriptive for 'Reset' button..zip](https://github.com/microsoft/terminal/files/7779574/Accessible.name.is.not.descriptive.for.Reset.button.zip)

### Expected Behavior

Accessible name should be more descriptive for 'Reset' button.
i.e. Screen Reade should announce 'Reset tab color button'.

### Actual Behavior

Accessible name is not descriptive for 'Reset' button.
i.e. Screen Reade is only announcing 'Reset button'.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
